src/lava/utils/float2fixed.py

```diff
@@ -1,26 +1,719 @@
+            p-quantile of the domain of a monitored variable to set largest and
+            smallest value of the dynamic rep_range of that variable. The
+            dynamic range will be used to determine the optimal floating- to
+            fixed-point conversion. The first entry must be smaller than the
+            second. 0 resp. 1 correspond to the minimum resp. maximum of the
+            domain.
+        find_connected_procs : bool
+            Find all Processes connected to passed Process
+        """
+        self.num_steps = num_steps
+        self.quantiles = quantiles
+        self._set_procs(proc=proc, find_connected_procs=find_connected_procs)
+        # Infer Variables and Ports of Processes.
+        self.var_ports = self._get_var_ports()
+        # If hierarchical Processes are involved, set up copy of model where
+        # Subprocesses are made explicit.
+        if self.hierarchical_procs:
+            self._explicate_hierarchical_procs()
+        else:
+            pass
+        # Get fixed-point ProcessModels of Processes.
+        self.fixed_pt_proc_models = self._get_fixed_pt_proc_models()
+        # Get fixed-point ProcessModels of Processes.
+        self.fixed_pt_proc_models = self._get_fixed_pt_proc_models()
+        # Fetch conversion data from fixed-point ProcessModel.
+        self.conv_data = self._get_conv_data()
+        # Instantiate monitors for recording from dynamic variables.
+        self.monitors = self._create_monitoring_infrastructure()
+        # Run floating-point model and store activations monitored variables.
+        self._run_procs()
+        # Update scale domains for scaling of parameters.
+        self.scale_domains = self._update_scale_domains()
+        # Get the factor/ for the scaling functions for parameter conversion.
+        self.scaling_factors = self._find_scaling_factors()
+        # Scale parameter from a floating- to a fixed-point representation.
+        self.scaled_params = self.scale_parameters()
+
+    def _set_procs(self, proc, find_connected_procs=True) -> None:
+        """Set list of Lava Processes for conversion. Either pass a Lava
+        Process and find connected Lava Processes or pass list of Lava
+        Processes.
+        Write Processes p in a dictionary with structure {p.id : p}.
+
+        Parameters
+        ----------
+
+        proc : AbstracProcess or List[AbstractProcess]
+            Processes to be converted
+        find_connected_procs : bool
+            Find all Processes connected to passed Process
+        """
+        if issubclass(type(proc), AbstractProcess):
+            if find_connected_procs:
+                proc_list = find_processes(proc)
+            else:
+                proc_list = [proc]
+        elif isinstance(proc, list):
+            for p in proc:
+                if not issubclass(type(p), AbstractProcess):
+                    raise TypeError("list 'proc' must contains Processes")
```

Comment:
"contain"

Reply:
Addressed in 58fd18a6c5ec776b81b991a6260f4edcc2d2ca2d.